docs/understand/hardware_implementation.rst

```diff
@@ -97,3 +103,67 @@ Scalar Unit
 The scalar unit performs instructions that are uniform within a warp. It
 thereby improves efficiency and reduces the pressure on the vector ALUs and the
 vector register file.
+
+CDNA Architecture
+===============================================================================
+
+The general structure of the compute units stays mostly the same as in GCN
+architectures. The most prominent change is the addition of matrix ALUs, that
+can greatly improve the performance of algorithms involving matrix
+multiply-accumulate operations for
+:doc:`int8, float16, bfloat16 or float32<rocm:about/compatibility/precision-support>`.
+
+.. _cdna3_cu:
+
+.. figure:: ../data/understand/hardware_implementation/cdna3_cu.png
+  :alt: Block diagram showing the structure of a CDNA3 compute unit. It includes
+        Shader Cores, the Matrix Core Unit, a Local Data Share used for sharing
+        memory between threads in a block, an L1 Cache and a Scheduler. The
+        Shader Cores represent the vector ALUs and the Matrix Core Unit the
```

Comment:
A verb is missing from the end.
i.e.: "ALUs and the Matrix Core Unit shows the"

Reply:
This was implicit, and meant that "the Matrix Core Unit [represents] the matrix ALUs". Wrote it out explicitly.